src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentMetadataAsSourceFileProvider.cs

```diff
@@ -118,6 +121,7 @@ internal sealed class PdbSourceDocumentMetadataAsSourceFileProvider : IMetadataA
             var isReferenceAssembly = MetadataAsSourceHelpers.IsReferenceAssembly(symbol.ContainingAssembly);
             if (isReferenceAssembly)
             {
+                telemetryMessage.SetReferenceAssembly("yes");
```

Comment:
not sure I understand the "yes" vs "resolved" here. Is this because we can early return if we log "Symbol_found_in_assembly_path_0"?

Reply:
Sometimes its a reference assembly, and therefore there is nothing we can do about it. Sometimes its a reference assembly, but we can work out where the implementation is, and provide a better experience for the user. We want to know when each happens.